### PR TITLE
must-gather: stop using tini in must gather helper pod

### PR DESCRIPTION
**Pull request by subhamkrai**
stop using tini in must gather helper pod template. This is the only
the place we are using tini.

Signed-off-by: subhamkrai <srai@redhat.com>

### CODE REVIEW

**Review by crombus — CHANGES_REQUESTED** [review]
Changes Looks good but commit msg should be updated.

In the second commit update the must-gather helper pod template not must-gather pod template.

**Comment by agarwal-mudit** [comment]
/hold
1. Requires review from @leseb 
2. Review commits need to be rectified

File: must-gather/templates/pod.template
Comment (leseb) [inline must-gather/templates/pod.template:19]: This change should be part of the first commit (removal of `tini`)

**Review by leseb — CHANGES_REQUESTED** [review]

**Review by leseb — APPROVED** [review]

**Pull request merged**
